packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -1275,6 +1275,29 @@ class PostsRepo extends AbstractRepo<"Posts"> {
     });
   }
 
+  async getUsersMostCommentedPostSince(userId: string, since: Date): Promise<{
+    post: DbPost,
+    commentCount: number,
+  } | null> {
+    return this.getRawDb().oneOrNone(`
+      -- PostsRepo.getUsersMostCommentedPostSince
+      SELECT
+        ROW_TO_JSON(p.*) "post",
+        COUNT(c.*) FILTER (WHERE c."postedAt" > $2) "commentCount"
```

Comment:
Could you add a filter for not deleted/draft etc (whatever the most standardised check for this is)?